bundles/org.openhab.binding.yamahamusiccast/src/main/java/org/openhab/binding/yamahamusiccast/internal/YamahaMusiccastHandler.java

```diff
@@ -1140,6 +1232,29 @@ private String makeRequest(@Nullable String topicAVR, String url) {
         return makeRequest("Volume", host + YAMAHA_EXTENDED_CONTROL + zone + "/setVolume?volume=" + value);
     }
 
+    /**
+     * Sets the volume in decibels (dB).
+     *
+     * @param value volume in dB (decibels)
+     * @param zone name of zone
+     * @param host hostname or ip address
+     * @return HTTP request
+     */
+    private @Nullable String setVolumeDb(float value, @Nullable String zone, @Nullable String host) {
+        if (value < Float.parseFloat(getThing().getConfiguration().get("volumeDbMin").toString())) {
```

Comment:
Consider introducing a local variable for the parsed values to avoid parsing twice when out of valid range.